guacamole/src/main/java/org/apache/guacamole/rest/APIRequest.java

```diff
@@ -58,6 +68,12 @@ public APIRequest(HttpServletRequest request,
 
         super(request);
 
+        // Grab the remote host info.
+        this.remoteHost = request.getRemoteHost();
+
+	// Grab the remote ip info.
```

Comment:
Ugh ... you monster.

I'll double-check that no further tabs exist in these changes.